crates/rooch-types/src/address.rs

```diff
@@ -44,24 +48,26 @@ pub trait RoochSupportedAddress:
 }
 
 /// Multi chain address representation
-/// The address is distinguished by the coin_id type, coin_id type standard is defined in [slip-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
+/// The address is distinguished by the multichain id type, multichain id type standard is defined in [slip-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
 pub struct MultiChainAddress {
-    pub coin_id: CoinID,
+    pub multichain_id: RoochMultiChainID,
     pub raw_address: Vec<u8>,
 }
 
 impl MultiChainAddress {
-    pub(crate) fn new(coin_id: CoinID, raw_address: Vec<u8>) -> Result<Self> {
+    pub(crate) fn new(multichain_id: RoochMultiChainID, raw_address: Vec<u8>) -> Result<Self> {
         Ok(Self {
-            coin_id,
+            multichain_id,
             raw_address,
         })
     }
 
     pub fn is_rooch_address(&self) -> bool {
-        self.coin_id == CoinID::Rooch
+        self.multichain_id.multichain_id().id() == BuiltinChainID::Dev.chain_id().id()
```

Comment:
Rooch only has one multichain_id. We do not need to distinguish dev|test|main in the multichain_id scenario.

Reply:
Resolved to `is_rooch`.